src/main/java/com/lorepo/icplayer/client/module/choice/ChoicePrintable.java

```diff
@@ -15,9 +15,13 @@ public ChoicePrintable(ChoiceModel model) {
 		this.model = model;
 	}
 	
-	public String getPrintableHTML(boolean showAnswers) {
+	public String getPrintableHTML(String className, boolean showAnswers) {
 		if (model.getPrintable() == PrintableMode.NO) return null;
 		
+		if (className.length() > 0) {

```

Comment:
DRY  - można by to wyciągnąć do jakiegos utila/serwisu, dodatkowo gdyby pojawiła się potrzeba dodania dodatkowych klas, to będzie łatwiej modyfikować to w jednym miejscu.